src/Utils.js

```diff
@@ -76,7 +73,7 @@ class Utils {
    * @returns {Promise}
    */
   static ensureDir(dirPath) {
-    return ENSURE_DIR_P(dirPath);
+    return fsp.mkdir(dirPath, {recursive: true, mode: 0o700});
```

Comment:
replacing fs-extra with native equivalent.

note: works fine when the dir already exists:
> Calling fsPromises.mkdir() when path is a directory that exists results in a rejection only when recursive is false.

source: https://nodejs.org/api/fs.html#fspromisesmkdirpath-options